.env

```diff
@@ -1 +1,4 @@
+CUTTLE_ENV=dev
```

Comment:
Moved to a new `CUTTLE_ENV` env variable which makes more sense since it's not going to put sails in actual prod mode, but lets us do prod-like stuff.